.golangci.yml

```diff
@@ -36,6 +36,7 @@ linters-settings:
         arguments:
           - fmt.Printf
           - fmt.Println
+          - fmt.Fprint
```

Comment:
If we want this change here, we probably want it in [baton-template](https://github.com/ConductorOne/baton-template).